ReportUtilityBills/resources/util.rb

```diff
@@ -98,6 +101,12 @@ def self.simple(fuel_type, header, fuel_time_series, is_production, rate, bill,
       if net_elec < 0 # net producer, give credit at user specified rate
         bill.annual_production_credit += -net_elec * rate.net_metering_user_excess_sellback_rate
       end
+      if (bill.annual_production_credit > bill.annual_energy_charge) || (net_elec < 0)
```

Comment:
Why is this not just `if bill.annual_production_credit  != 0`? I don't really understand what is happening here.

Reply:
I tried to pull the two terms from directly above: `bill.annual_production_credit > bill.annual_energy_charge` and `net_elec < 0`. If you had (previously) entered either of these blocks, move all annual production credit to the final month of the year.

Reply:
I'll try your suggested simplification.